### DISCUSSION TITLE
any $cyan alternative color

### DISCUSSION THREAD

**Discussion by danny007in** [twbs/bootstrap#34039] [Q&A]
any $cyan alternative color ,
because in `color-yiq` it was returning white,
but in `color-contrast` returning black.

so which kind for `$cyan` i needed to use,
to return white in `color-contrast`

**Comment by ffoodd** [comment]
V5 raised up the minimum contrast ratio to improve accessibility. Cyan and light blues in general should be used with a dark color.

I suggest you darken your cyan, or try to change its hue.
